src/Accounts/Accounts/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 * Updated `Connect-AzAccount` to accept parameter `MaxContextPopulation` [#9865](https://github.com/Azure/azure-powershell/issues/9865)
+* Update SubscriptionClient version to 2019-06-01 and display tenant domains [#9838](https://github.com/Azure/azure-powershell/issues/9838)
```

Comment:
[#9838] is sufficient. Remove link of it. I will correct Line 21 during release.